### DISCUSSION TITLE
Error en copilot

### DISCUSSION THREAD

**Discussion by danny27475** [community/community#10474] [Copilot Conversations] [answered]
[INFO] [auth] [2022-01-24T03:44:40.270Z] Invalid copilot token: missing token: 404 
[ERROR] [default] [2022-01-24T03:44:40.300Z] GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

**Comment by davecheney** [comment, marked as answer]
This error indicates that you are on the waitlist. I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
